src/porepy/geometry/distances.py

```diff
@@ -178,20 +193,25 @@ def dot(v1, v2):
     return np.sqrt(np.sum(np.power(dist, 2), axis=0)), cp1, cp2
 
 
-def points_segments(p, start, end):
+def points_segments(
+    p: np.ndarray, start: np.ndarray, end: np.ndarray
+) -> tuple[np.ndarray, np.ndarray]:
     """Compute distances between points and line segments.
 
     Also return closest points on the segments.
```

Comment:
Maybe write something along the line of "The function also returns the closest points on the segments" here

Reply:
Done.